### PR TITLE
format FOLIO driver getMyProfile expiration_date

### PR DESCRIPTION
**Pull request by ihardy**

### CODE REVIEW

File: module/VuFind/src/VuFind/ILS/Driver/Folio.php [hunk 1]
```diff
@@ -625,7 +625,9 @@ public function getMyProfile($patron)
             'zip' => $profile->personal->addresses[0]->postalCode ?? null,
             'phone' => $profile->personal->phone ?? null,
             'mobile_phone' => $profile->personal->mobilePhone ?? null,
-            'expiration_date' => $profile->expirationDate ?? null,
+            'expiration_date' => $this->dateConverter->convertToDisplayDate(
+                "Y-m-d H:i", $profile->expirationDate
+            ) ?? null,
```
Comment (demiankatz) [inline module/VuFind/src/VuFind/ILS/Driver/Folio.php:630]: I don't think `?? null` works here... that's to ensure PHP doesn't throw an "undefined" error about `$profile->expirationDate`, but now the code is assuming that `$profile->expirationDate` will be set. Might be cleaner to do:

<pre>
$expiration = isset($profile->expirationDate)
    ? $this->dateConverter->convertToDisplayDate("Y-m-d H:i", $profile->expirationDate)
    : null;
</pre>

And then assigning the `$expiration` value in the return array...

Comment (ihardy) [reply module/VuFind/src/VuFind/ILS/Driver/Folio.php]: Thanks! applied this suggestion.

**Comment by demiankatz** [comment]
Thanks, @ihardy, looks good to me. I'm merging now! (I just made a trivial edit to adjust the indentation of the ternary operator for improved readability... though apparently that's just my personal preference because the style checkers are happy either way).

**Comment by ihardy** [comment]
Thanks Demian, wasn't my first preference either but the linter wasn't happy with what I had, so glad you found another way to do it.

**Pull request merged**